docker-dev-setup/README.md

```diff
@@ -0,0 +1,97 @@
+# Docker container based development setup
+
+This setup uses Docker to run the app and the database,
+therefore it is very easy to set up on any platform that supports Docker (Linux, Windows via WSL2, etc.).
+This setup is also recommended if you don't want to install PHP and other dependencies on your host machine,
+or if you already have a different (incompatible) version of PHP installed.
+
+## Requirements
+
+- [Docker](https://docs.docker.com/engine/)
+- [Docker Compose](https://docs.docker.com/compose/)
```

Comment:
Is Docker Compose a requirement? AFAIK newer Docker Engine versions come bundled with the `docker compose` command by default. If it is not available, they should update their Docker installation,

Reply:
We can probably say that this requires Docker Engine version 25.0.1 or newer. In that case `docker compose` should most definitely work.

Reply:
Docker Compose is a requirement. But you are right, it doesn't need to be manually installed. What do you suggest?

Reply:
I have created a PR for your PR which addresses the issue: https://github.com/Trigary/mars/pull/1